.gitignore

```diff
@@ -0,0 +1,12 @@
+
+admin/regEmp.php
+
+admin/controller/empAjaxValidation.php
+
+admin/regEmp.php
+
+phpinfo.php
+
+untitled.sql
+
+admin/regEmp.php
```

Comment:
Why are we gitignoring any of these files?

Reply:
Like the commit message for these says, they're "bad files"

Reply:
what does that even mean hahahahah